pages/files-table.module.scss

```diff
@@ -133,15 +125,17 @@
   align-items: center;
   justify-content: space-between;
   width: 100%;
-  position:absolute;
-  top:50%;
+  position: absolute;
+  top: 50%;
```

Comment:
does this position absolute do anything? you have a translation on it, is it even necessary?